### ISSUE TITLE
Poly Loss image segmentation

### ISSUE THREAD

**Issue by Ehteshamciitwah** [yiyixuxu/polyloss-pytorch#3]
Hello, Thank you for your work. I want to use the poly loss for image segmentation. I have input [B,C,H,W] and mask as [B,H,W]. I have  ignore_indexes too in masks. how can I use your provided loss function correctly? Thank you

**Comment by yiyixuxu** [comment]
Hi:

1. `ignore_indexes` is not supported here. 
2.  if you have input `[B,C,H,W]` and mask `[B,H,W] `- you can insert a channel dimension of size `1` in your mask, do `mask[:,None,...]` or `mask.unqueeze(1)`( learn more about `unsqueeze` here https://pytorch.org/docs/stable/generated/torch.unsqueeze.html)

**Comment by flower-me** [comment]
> 

Would you like to know if you have successfully applied image segmentation and how to do it? Thank you.

**Comment by yiyixuxu** [comment]
@flower-me here is an example where I applied on segmentation for medical images 

https://wandb.ai/yiyixu/BTCV/reports/PolyLoss-for-Multi-organ-Segmentation-Task--VmlldzoxOTk3Nzg1

notebook here: 
https://colab.research.google.com/gist/yiyixuxu/82e2878a6e688e167d2b665928e41cc7/polyloss_unetr_btcv_segmentation_3d_w-b.ipynb

**Comment by flower-me** [comment]
First of all, thank you very much for your help, but I can't get the link to this notebook. Can you share it again






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yiyixuxu/polyloss-pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月7日(星期二) 凌晨0:40
***@***.***&gt;;
抄送:&nbsp;"杨海红 ***@***.******@***.***&gt;;
主题:&nbsp;Re: [yiyixuxu/polyloss-pytorch] Poly Loss image segmentation (Issue #3)





 
@flower-me here is an example where I applied on segmentation for medical images
 
https://wandb.ai/yiyixu/BTCV/reports/PolyLoss-for-Multi-organ-Segmentation-Task--VmlldzoxOTk3Nzg1
 
notebook here:
 https://colab.research.google.com/gist/yiyixuxu/82e2878a6e688e167d2b665928e41cc7/polyloss_unetr_btcv_segmentation_3d_w-b.ipynb
